### PR TITLE
Update the latest URLs

### PR DESCRIPTION
**Pull request by jsngyoon**
### Change Description ###

- Update the Tizen 6.5 M2 URLs

### CODE REVIEW

**Review by arisha-srbd — COMMENTED** [review]
Hello @jsngyoon. Hope you are doing well. 
I see that you have made some changes to update the URLs, so I wanted to know if you want me to review the whole PR or just check the updated parts. I shall do accordingly. Thank you. :)

File: docs/platform/developing/creating.md [hunk 1]
```diff
@@ -27,35 +27,35 @@ Image creation requires a kickstart file that describes how to create an image.
...
 
      - Tizen: latest: Unified / standard / `mobile-wayland-armv7l-tm1.ks`
 
        ```
-       $ wget http://download.tizen.org/releases/milestone/tizen/unified/tizen-unified_20210513.3/images/standard/mobile-wayland-armv7l-tm1/tizen-unified_20210513.3_mobile-wayland-armv7l-tm1.ks
+       $ wget http://download.tizen.org/releases/milestone/tizen/unified/tizen-unified_20211014.1/images/standard/mobile-wayland-armv7l-tm1/tizen-unified_20211014.1_mobile-wayland-armv7l-tm1.ks
        ```
 
-2. Modify the original kickstart file to include locally built RPMs into the Tizen image.
+2. Modify the original kickstart file to update the 'baseurl' properties and to include locally built RPMs into the Tizen image.
```
Comment (arisha-srbd) [inline docs/platform/developing/creating.md:41]: I think the word ‘baseurl’ should be under code tags to maintain consistency with the representation of ‘baseurl’ in **Line 64**. Please look into this.  
```suggestion
2. Modify the original kickstart file to update the `baseurl` properties and to include locally built RPMs into the Tizen image.
```

File: docs/platform/developing/creating.md [hunk 2]
```diff
@@ -27,35 +27,35 @@ Image creation requires a kickstart file that describes how to create an image.
...
 
    Add a --priority option to increase a local repositorie's priority for build.
```
Comment (arisha-srbd) [inline docs/platform/developing/creating.md:43]: The word “repository” has been spelled wrong here. 
```suggestion
   Add a --priority option to increase a local repository's priority for build.
```

File: docs/platform/developing/creating.md [hunk 3]
```diff
@@ -27,35 +27,35 @@ Image creation requires a kickstart file that describes how to create an image.
...
    For example: Tizen: latest: Unified / standard / `mobile-wayland-armv7l-tm1.ks`
```
Comment (arisha-srbd) [inline docs/platform/developing/creating.md:44]: Line 44: I think this line should be represented as an unordered list to maintain consistency with the representation in Line 33.

**Review by arisha-srbd — COMMENTED** [review]
@jsngyoon hello! Hope you are doing well. I have reviewed the whole PR and made some in-line suggestions. Additionally, I found some issues that I am not being able to comment on. I am listing down the correction suggestions:

1.	Line 1: “Create Tizen images with MIC”
Suggestion: “Create Tizen Images with MIC”
Reason: First headings should follow the title case. 

2.	Line 17: “create an image in a specific format”
Suggestion: “Create an image in a specific format.”
Reason: Each list should start with a capital letter and end with a period. Ref: https://developers.google.com/style/lists#capitalization-and-end-punctuation

3.	Line 18: “chroot into an image”
Suggestion: “Chroot into an image.”
Reason: Each list should start with a capital letter and end with a period. Ref: https://developers.google.com/style/lists#capitalization-and-end-punctuation

4.	Line 19: “convert an image to another format”
Suggestion: “Convert an image to another format.”
Reason: Each list should start with a capital letter and end with a period. Ref: https://developers.google.com/style/lists#capitalization-and-end-punctuation

5.	Line 25: “…To prepare the kickstart file:”
Suggestion: “…To prepare the kickstart file, please carry out the following steps:”
Reason: Lists should be introduced with a complete sentence. Ref: https://developers.google.com/style/lists#introductory-sentences-for-lists

6.	Line 65: “.......MIC uses the packages that exist in the local repository with a higher priority, when packages are available in both remote and local repositories.”
Suggestion: “........MIC uses the packages that exist in the local repository with a higher priority when packages are available in both remote and local repositories.”
_Reason: The comma after the word “priority” doesn’t seem necessary here._ 

7.	Line 66: “To add new packages into a Tizen image, add the new packages' names into the `%package` section, and add them into the `local` repository.”
Suggestion: “To add new packages into a Tizen image, add the new packages' names into the `%package` section, and add them to the `local` repository.”
Reason: The appropriate preposition should be “to” here. 

8.	Line 67: “You can check package's name in the `Name` section of the spec file.”
Suggestion: “You can check the package's name in the `Name` section of the spec file.”
Reason: An article missing here. 

9.	Line 71: “To create a Tizen image:”
Suggestion: “To create a Tizen image, follow the below steps:”
Reason: Lists should be introduced with a complete sentence. Ref: https://developers.google.com/style/lists#introductory-sentences-for-lists

**Comment by jsngyoon** [comment]
Hello, @arisha-srbd, thank you for your kind suggestions, but please review the changes I've made only.

**Review by arisha-srbd — COMMENTED** [review]
@jsngyoon hello. I have already made suggestions on the updated parts in my previous review. You can incorporate those only then. Let me know if there is any confusion, Thank you.

**Comment by jsngyoon** [comment]
Hello, @arisha-srbd, I've incorporated the related suggestions only as you requested.

**Review by arisha-srbd — APPROVED** [review]
@jsngyoon thank you for incorporating the changes in the updated parts. 

@asm-foysal @mijin85cho I am approving this PR for merging.

**Pull request merged**